### ISSUE TITLE
Attention mask error in certain models

### ISSUE THREAD

**Issue by danijel3** [jonatasgrosman/huggingsound#18]
Some models use attention mask, but some don't. For example, running HuggingSound with the Facebook Voxpopouli models (eg. `facebook/wav2vec2-base-10k-voxpopuli-ft-pl`)  crashes with the error that the attention mask cannot be found.

The solution is to change the relevant code [here](https://github.com/jonatasgrosman/huggingsound/blob/939162daeca81bc2ef0be664133998a6edcb5f75/huggingsound/speech_recognition/model.py#L113):

```
         with torch.no_grad():
              logits = self.model(inputs.input_values.to(self.device), attention_mask=inputs.attention_mask.to(self.device)).logits
```

We just need to add a simple check:

```
            with torch.no_grad():
                if 'attention_mask' in inputs:
                    logits = self.model(inputs.input_values.to(self.device),attention_mask=inputs.attention_mask.to(self.device)).logits
                else:
                    logits = self.model(inputs.input_values.to(self.device)).logits
```

I'm too lazy to make a PR now, so if someone could do it that would be swell :-)

**Comment by jonatasgrosman** [comment]
Hi @danijel3!
I've just fixed this issue. Thanks for debugging this :)
